### PR TITLE
Android View Appointments & Cancel Appointment

### LINKED ISSUE #204: Mobile Frontend: View & Cancel Appointments
## 📜 Description

- Allows the customer to view and cancel their appointments
- `View Appointments` page from [our website](https://lit-fortress-63303.herokuapp.com/#/):

image

## **✅ To do:**
- [x] Implement mobile frontend 
working on `android-cancelAppointment` branch

## **🆘 If need help for the issue you’re working on:**

**1) Write** comments in this issue 
**2) Add** label image
**3) Copy** the issue link and **paste** it to Discord's `#🐛issues` channel
**4)** When solved, **strike** the message in `#🐛issues`, **mark it** as `(SOLVED)`  + **remove** label `help wanted` + **write** comments in this issue

### PR DESCRIPTION
**Pull request by sandyl289**
- `View Appointments` & `Cancel Appointment` pages (must be logged in as customer)
- Everything works!!

Thanks Tyler & Jacek for helping me!!!

# View Appointments
image

# Cancel Appointments

## Attempt to cancel last appointment
image
image
image

## Attempt to cancel invalid appointment (not 7 days ahead) (works)
image
image

### CODE REVIEW

File: RepairShop-Android/app/src/main/java/ca/mcgill/ecse321/repairshop/BaseActivity.java [hunk 1]
```diff
@@ -43,8 +43,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
             return true;
         } else if (id == R.id.menuViewAppointments) {
             // Switch to View Appointments
+            startActivity(new Intent(this, ViewAppointments.class));
             return true;
-        } else if (id == R.id.menuLogout) {
+        } else if (id == R.id.menuCancelAppointment) {
+            // Switch to View Appointments
```
Comment (T-Syme) [inline RepairShop-Android/app/src/main/java/ca/mcgill/ecse321/repairshop/BaseActivity.java]: Tiny detail (feel free to ignore): comment should be for cancelling appointment

**Review by T-Syme — APPROVED** [review]
Looks good to me!

Ideally the .idea files shouldn't be there, but not a big deal! I wouldn't worry about it

File: RepairShop-Android/app/src/main/java/ca/mcgill/ecse321/repairshop/BaseActivity.java (hunk 1 above)
Comment (sandyl289) [reply RepairShop-Android/app/src/main/java/ca/mcgill/ecse321/repairshop/BaseActivity.java]: Thanks! 
Done: https://github.com/McGill-ECSE321-Winter2021/project-group-09/pull/215/commits/7ba78082b3ccb785b96ac401b046347cf491a318

**Review by zhannaklimanova — APPROVED** [review]

**Review by Shaswata79 — APPROVED** [review]

**Comment by sandyl289** [comment]
- Remove Titles 
image
image
image
image
image

image

**Pull request merged**